installer/create-cluster-eks.sh

```diff
@@ -0,0 +1,88 @@
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Usage:
+# create-cluster-eks.sh [EKS-CLUSTER-NAME] [AWS_DEFAULT_REGION]
+if [ $# -ne 2 ]; then
+  echo "Not enough arguments supplied"
+  echo "Usage: create-cluster-eks.sh [EKS-CLUSTER-NAME] [AWS_DEFAULT_REGION]"
+  exit 1
+fi
+
+export EKS_CLUSTER_NAME=$1
+export AWS_DEFAULT_REGION=$2
+
+# Create cluster
+# https://docs.aws.amazon.com/eks/latest/userguide/create-cluster.html
+ZONE_1A=a
+ZONE_1B=b
+ZONE_1D=d
+eksctl create cluster \
+  --name "$EKS_CLUSTER_NAME" \
+  --version 1.14 \
+  --region "$AWS_DEFAULT_REGION" \
+  --nodegroup-name standard-workers \
+  --node-type t3.medium \
+  --nodes 3 \
+  --nodes-min 1 \
+  --nodes-max 7 \
+  --managed \
+  --zones="$AWS_DEFAULT_REGION$ZONE_1A,$AWS_DEFAULT_REGION$ZONE_1B,$AWS_DEFAULT_REGION$ZONE_1D"
+
+# Create nodegroup for Strimzi resources.
+# https://aws.amazon.com/premiumsupport/knowledge-center/eks-multiple-node-groups-eksctl/
+eksctl create nodegroup \
+  --cluster="$EKS_CLUSTER_NAME" \
+  --name=kafka-pool-0 \
+  --region "$AWS_DEFAULT_REGION" \
+  --node-type r5.large \
+  --nodes 3 \
+  --managed \
+  --node-labels=dedicated=StrimziKafka
+
+# Create EFS
+aws efs create-file-system \
+  --creation-token "$EKS_CLUSTER_NAME" \
+  --performance-mode generalPurpose \
+  --throughput-mode bursting \
+  --region "$AWS_DEFAULT_REGION" \
+  --tags Key=Name,Value="$EKS_CLUSTER_NAME"
+
+sleep 10
+
+FILE_SYSTEM_ID="$(aws efs describe-file-systems --query "FileSystems[?Name=='$EKS_CLUSTER_NAME'].FileSystemId" --output json | jq -r '.[]')"
+echo "File system id: $FILE_SYSTEM_ID"
+
+SECURITY_GROUP_IDS="$(aws eks describe-cluster --name "$EKS_CLUSTER_NAME" | jq -r '.cluster.resourcesVpcConfig.securityGroupIds | .[0]')"
+echo "Security group id's: $SECURITY_GROUP_IDS"
+
+CLUSTER_SECURITY_GROUP_ID="$(aws eks describe-cluster --name "$EKS_CLUSTER_NAME" | jq -r '.cluster.resourcesVpcConfig.clusterSecurityGroupId')"
+echo "Cluster security group id: $CLUSTER_SECURITY_GROUP_ID"
+
+# Mount EFS targets (one for each zone)
+for zone_value in $ZONE_1A $ZONE_1B $ZONE_1D; do
+
+  ZONE="$(echo "$zone_value" | tr '[:lower:]' '[:upper:]')"
+  echo "Zone: $ZONE"
+
+  SUBNET_ID="$(aws ec2 describe-subnets --filters Name=tag:Name,Values="eksctl-$EKS_CLUSTER_NAME-cluster/SubnetPublicUSEAST1$ZONE" --output json | jq -r '.Subnets | .[].SubnetId')"
+  echo "Subnet id: $SUBNET_ID"
+
+  aws efs create-mount-target \
+    --file-system-id "$FILE_SYSTEM_ID" \
+    --subnet-id "$SUBNET_ID" \
+    --security-groups "$SECURITY_GROUP_IDS" "$CLUSTER_SECURITY_GROUP_ID"
```

Comment:
s/ "$CLUSTER_SECURITY_GROUP_ID"/"${CLUSTER_SECURITY_GROUP_ID//null/}" .
This will fail the cluster sc group is empty.

Reply:
>Am I also wondering if  `SECURITY_GROUP_IDS` can be null too, probably not.

correct

Reply:
@eschizoid could you fix this too, I faced this error.